src/ui/templates/shared-pages/single-radio.njk

```diff
@@ -27,43 +24,41 @@
     }) }}
   {% endif %}
 
+  {# if applicationAnswer is null or undefined, then should set submitted answer to submittedValues value #}
+  {% if applicationAnswer === null or applicationAnswer === undefined %}
+    {% set submittedAnswer = submittedValues[FIELD_ID] %}
+  {% else %}
+    {% set submittedAnswer = submittedValues[FIELD_ID] or applicationAnswer %}
+  {% endif %}
+
```

Comment:
I found this hard to read/follow, would this work?

```
{% if applicationAnswer %}
  {% set submittedAnswer = submittedValues[FIELD_ID] or applicationAnswer %}
{% else %}
  {% set submittedAnswer = submittedValues[FIELD_ID] %}
{% endif %}
```

If not maybe something like this?

```
{% set hasNoApplicationAnswer = applicationAnswer !== null or applicationAnswer !== undefined %}

{% if hasNoApplicationAnswer %}
  {% set submittedAnswer = submittedValues[FIELD_ID] %}
{% else %}
  {% set submittedAnswer = submittedValues[FIELD_ID] or applicationAnswer %}
{% endif %}
```

Just the "is null or is undefined" parts are strange so want to be as clear as possible 😊

Reply:
done